content/administration/maintain/google_oauth.rst

```diff
@@ -56,26 +68,25 @@ Edit app registration
 - Next, click on the :guilabel:`Save and Continue` button.
 - Then, skip the :guilabel:`Scopes` page by scrolling to the bottom and clicking on
   :guilabel:`Save and Continue`.
-- Next, add the email being configured under the :guilabel:`Test users` step by clicking on
-  :guilabel:`Add Users` and then the :guilabel:`Save and Continue` button. A summary of the
-  :guilabel:`App registration` appears.
+- Next, if continuing in testing mode (External), add the email addresses being configured under the
```

Comment:
Don't see the benefit of adding this part?
> Next, if continuing in testing mode (External)

The previous version seemed clearer because it was active instruction, and not an "if" scenario...

Reply:
This is a common mistake made during configuration and Google is currently changing up requirements so I was avoiding which Mode to put the configuration. I think it is important to include this part.